jsconfig.json

```diff
@@ -0,0 +1 @@
+{"compilerOptions":{"target":"es6","experimentalDecorators":true},"exclude":["node_modules","bower_components","tmp","vendor",".git","dist"]}
```

Comment:
Can you leave a note in the pull request why this file was added?

Reply:
Ugh, it shouldn't have been. I'll configure my global git ignore!